### PR TITLE
feat(vinnie): massive pizza change

### PR DESCRIPTION
**Pull request by Porthorian**
In light of certain events, I have decided to move pizza further up the chart.

These events entailed. Having an orgasmic buffalo chicken ranch pizza with pepperoni. After being to Chicago and learning how they make their deep-shit pizza @dean-haw. I have found a new love of bumping Pizza before all the other fish.

Also Pasta on Pizza is a sin. Anchovies on a Pizza is a sin.

### CODE REVIEW

File: vinnie.md [hunk 1]
```diff
@@ -15,9 +15,9 @@
 15. Pulled Pork
 16. Fried Rice
 17. BLT
-18. Sushi
-19. Shrimp
-20. Fish 
-21. Pizza
+18. Pizza
```
Comment (aanthonyrizzo) [inline vinnie.md:18]: I refuse to believe that BLT is higher than pizza but this change is a good step in the right direction.

**Review by aanthonyrizzo — APPROVED** [review]
I think there is still more work to be done but I believe in iteration

File: vinnie.md (hunk 1 above)
Comment (Porthorian) [reply vinnie.md:18]: Yeah needs further design discussions, BLT's are pretty good.

**Review by dean-haw — APPROVED** [review]
Now we're starting to make some sense Vin

**Comment by dean-haw** [comment]
Just want to make everyone aware though that Vinnie went to Chicago and DID NOT get a deep dish pizza. It's like going to Jersey asking for shrimp and grits or some shit.

**Pull request merged**